### PR TITLE
testing/stgit: new aport

### PR DESCRIPTION
**Pull request by jbenden**
http://www.procode.org/stgit/
Manage a stack of patches using GIT as a backend

Signed-off-by: Joseph Benden <joe@benden.us>

### CODE REVIEW

File: testing/stgit/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Contributor: Joseph Benden <joe@benden.us>
+# Maintainer: Joseph Benden <joe@benden.us>
+
+pkgname=stgit
+pkgver=0.19
+pkgrel=0
+pkgdesc="Manage a stack of patches using GIT as a backend"
+arch="noarch"
+url="http://www.procode.org/stgit/"
+license="GPL"
```
Comment (maxice8) [inline testing/stgit/APKBUILD]: ambiguous and not SDPX compliant

File: testing/stgit/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+options="!check !strip" # unit-tests fail due to iconv usage
+depends="git python3"
+makedepends="python3 asciidoc xmlto perl"
```
Comment (maxice8) [inline testing/stgit/APKBUILD]: python3 is in depends=, doesn't need to be in makedepends

File: testing/stgit/APKBUILD (hunk 1 above)
Comment (jbenden) [reply testing/stgit/APKBUILD]: Fixed. Thank you for the review!

File: testing/stgit/APKBUILD (hunk 2 above)
Comment (jbenden) [reply testing/stgit/APKBUILD]: Fixed. Thanks!

File: testing/stgit/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+pkgdesc="Manage a stack of patches using GIT as a backend"
+arch="noarch"
+url="http://www.procode.org/stgit/"
+license="GPL-2.0-or-later"
+options="!check !strip" # unit-tests fail due to iconv usage
+depends="git python3"
+makedepends="asciidoc xmlto perl"
+subpackages="$pkgname-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/ctmarinas/stgit/archive/v${pkgver}.tar.gz"
+builddir="$srcdir/${pkgname}-$pkgver"
```
Comment (maxice8) [inline testing/stgit/APKBUILD]: default value, can be removed

**Pull request closed without merging**